src/ngraph/runtime/cpu/mkldnn_utils.cpp

```diff
@@ -639,7 +639,13 @@ bool runtime::cpu::mkldnn_utils::is_mkldnn_padded_layout(const mkldnn::memory::d
 bool runtime::cpu::mkldnn_utils::use_mkldnn_kernel(const ngraph::Node* node)
 {
     auto op_annotations = static_cast<const ngraph::op::Op*>(node)->get_op_annotations();
-    return (op_annotations &&
-            static_pointer_cast<ngraph::runtime::cpu::CPUOpAnnotations>(op_annotations)
-                ->is_mkldnn_op());
+    if (op_annotations)
```

Comment:
Since all op annotations will be created in the backend, we should be able to statically cast to CPUOpAnnotations.